docs/area-owners.md

```diff
@@ -2,34 +2,43 @@
+| **[area-security](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-security)**                 | [@joperezr](https://github.com/joperezr) [@mikekistler](https://github.com/mikekistler) [@halter73](https://github.com/halter73) [@mackinnonbuck](https://github.com/mackinnonbuck)                   | security features/work                                                                                                                                                 |
+| **[area-signalr](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-signalr)**                   | [@adityamandaleeka](https://github.com/adityamandaleeka) [@mikekistler](https://github.com/mikekistler) [@brennanconroy](https://github.com/brennanconroy)                                            | SignalR clients and servers                                                                                                                                            |
+| **[area-ui-rendering](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-ui-rendering)**         | [@joperezr](https://github.com/joperezr) [@danroth27](https://github.com/danroth27) [@mackinnonbuck](https://github.com/mackinnonbuck)                                                                | MVC Views/Pages, Razor Views/Page                                                                                                                                      |
+| **[area-unified-build](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-unified-build)**       | [@joperezr](https://github.com/joperezr) [@wtgodbe](https://github.com/wtgodbe)                                                                                                                       | Work related to https://github.com/dotnet/dotnet                                                                                                                       |
+
+| other repo          | Owners                                                                                                                                                   | Description                                                                                  |
+|---------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------|
+| {aspire}            | [@joperezr](https://github.com/joperezr) [@maddymontaquila](https://github.com/maddymontaquila) and various                                              | [https://github.com/dotnet/aspire](https://github.com/dotnet/aspire)                         |
```

Comment:
"and various others" isn't very useful. Suggest removing that (for this and the row below)